app/code/Magento/Wishlist/Controller/Index/Plugin.php

```diff
@@ -30,26 +30,34 @@ class Plugin
     protected $config;
 
     /**
-     * @var \Magento\Framework\App\Response\RedirectInterface
+     * @var \Magento\Framework\Message\ManagerInterface $messageManager
      */
     protected $redirector;
 
+    /**
+     * @var \Magento\Framework\Message\ManagerInterface $messageManager
+     */
+    protected $messageManager;
```

Comment:
Please use only `private` properties for core code as usage of inheritance is not encouraged.

Reply:
Done.